cmd/result.go

```diff
@@ -90,11 +90,18 @@ func shouldLog(err int) (members []string) {
 	return
 }
 
-func (res *Result) allowedCaps() (allowed map[CapabilityV1]string) {
+func (res *Result) allowedCaps(container ContainerV1) (allowed map[CapabilityV1]string) {
+
 	allowed = make(map[CapabilityV1]string)
 	for k, v := range res.Labels {
-		if strings.Contains(k, "audit.kubernetes.io/allow-capability-") {
-			capName := strings.Replace(strings.ToUpper(strings.TrimPrefix(k, "audit.kubernetes.io/allow-capability-")), "-", "_", -1)
+		if strings.Contains(k, "audit.kubernetes.io/pod/allow-capability-") {
```

Comment:
Could you please add a test for this code :) One that goes through both cases here.

Reply:
Done